doc/dune-files.rst

```diff
@@ -668,6 +668,12 @@ Executables can also be linked as object or shared object files. See
   pulled in. This field is available since the 2.0 version of the dune
   language.
 
+- ``(embed_in_plugin_libraries <library-list>)`` especifies a list of libraries
+  to link statically when using ``plugin`` linking mode. By default, no
+  libraries are linked in.  Any library which is a dependency of the executable
+  that is an inverse dependency of one the libraries in ``<library-list>`` will
+  also be statically linked.
```

Comment:
This wording is a bit confusing to me. 

> Any library which is a dependency of the executable

Dependency in the compilation or linking sense?

> that is an inverse dependency of one the libraries in ``<library-list>`

So if we have:

```
(libraries foo baz)
(embed_in_plugin_libraries bar)
```

and `foo` depends on `bar,` we should link in `foo` and `bar`?

What about the transitive dependencies of `bar`?

Reply:
I updated the doc according to the new, simpler implementation.